python/gps/algorithm/algorithm_mdgps.py

```diff
@@ -40,24 +51,29 @@ def iteration(self, sample_lists):
         Args:
             sample_lists: List of SampleList objects for each condition.
         """
-        # Store the samples and evaluate the costs.
+        # Store the samples.
         for m in range(self.M):
             self.cur[m].sample_list = sample_lists[m]
-            self._eval_cost(m)
 
-        # Update dynamics linearizations.
-        self._update_dynamics()
-
-        # On the first iteration, need to catch policy up to init_traj_distr.
+        # On the first iteration, need to train on init_traj_distr.
         if self.iteration_count == 0:
-            self.new_traj_distr = [
-                self.cur[cond].traj_distr for cond in range(self.M)
-            ]
+            self.new_traj_distr = [self.cur[m].traj_distr \
```

Comment:
the previous code here is actually better style, according to pylint. (pylint doesn't like "\")